extensions/vertx/runtime/pom.xml

```diff
@@ -47,6 +47,10 @@
             <groupId>io.vertx</groupId>
             <artifactId>vertx-rx-java2</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-smallrye-reactive-streams-operators</artifactId>
+        </dependency>
```

Comment:
Why does the vert.x extension depend on SR-RSO now? I don't see any new code in that extension.

Reply:
It's to have access to the Axle classes. It's was merged in another PR. It should be fixed with the new Axle stuff.

Reply:
I don't understand how SR-RSO is related to Axle.

Reply:
Axle provides methods returning `PublisherBuilder` defined in RSO.

Reply:
Well, OK, sorry, but does the vert.x extension use `PublisherBuilder` now?

Reply:
It exposes the Axle Vert.x instance so somehow yes...
This dependency was added to be more convenient (in another PR, this is just a fix as the dependency for not declared in the runtime).

Reply:
If it's exposed by the Axle Vert.x instance, it should be imported by the vertx axle module, no? If it's for convenience, that's another thing.

Reply:
Vert.x Axle depends on the spec only, not on the implementation. Here we explicitly depend on the implementation.